webpack.config.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require ('webpack')
+
+module.exports = {
+    entry: './source/likely.js',
+    output: {
+        filename: 'likely.js',
+        library: 'likely',
+        libraryTarget: 'umd',
+    },
+    plugins: [
+        new webpack.optimize.OccurrenceOrderPlugin(),
+        new webpack.optimize.DedupePlugin(),
+        new webpack.optimize.UglifyJsPlugin(),
```

Comment:
We could optimize this a little bit more by setting the `screw-ie8` option to `true`. (It might be called a bit differently; can’t find the proper name from the phone, sorry.)